lib/readline.js

```diff
@@ -1331,7 +1331,8 @@ function codePointAt(str, index) {
   }
   return code;
 }
-exports.codePointAt = codePointAt;
+exports.codePointAt = util.deprecate(codePointAt,
+    'codePointAt() is deprecated. Use String.prototype.codePointAt');
```

Comment:
I guess that would a bit irresponsible. People use repl to discover stuff. We need a roadmap to avoid these in the future